### PR TITLE
Created Logo Designs #217 - Vector version of the logo added into the "logo" folder and in the README

### PR DESCRIPTION
**Pull request by RAFAELDEV2016**
Hi,

I have turned the SVG version of the logo prepared by Sabihashaik into a true vector SVG version of the logo.
I have put the file into a new floder named "logo".
I have included the logo into the README.MD file.

I hope it will suit you.

Best regards

### CODE REVIEW

**Comment by RAFAELDEV2016** [comment]
Sorry, I made a mistake and closed the pull request.
I send it again.

**Comment by juliogonzalez** [comment]
@RAFAELDEV2016 looks good, but some questions (and suggestions).

I am not an design expert at all, so maybe some questions are stupid.

Not that important:
- Why are there only two layers? One contains one of the `TDS` "tags", and the other everything else. So at least with inkscape I can't resize or elements separately. Shouldn't each element be a layer?

Important:
- There's a a white box on the top that doesn't allow full transparency.
- Most likely the two `TDS` tags should have the same color and "mirrored" in terms of position (or in other words, FMPOV the black tag should move under the server and have the same color as the other thag)
- Is it normal that the image is blurry? It's not that evident with the default size (200x200) but it becomes more evident (specially for the tags) if I try to scale with inkscape:
image

@GeoffMontee can you also provide feedback?

**Comment by RAFAELDEV2016** [comment]
Hello @juliogonzalez 
I have taken into account all your requests.
Could you please check and confirm that it is ok ?

**Comment by RAFAELDEV2016** [comment]
Ping @juliogonzalez could you please have a look at pull request #274 and tell me if you can approve and merge please ?
All your requests should have been taken into account.

**Review by juliogonzalez — APPROVED** [review]
Sorry about the delay, lately I can only review tds_fdw stuff during weekends.

FMPOV the current logo on the PR looks good! Thanks a lot!

My only comment is, can we maybe change the filename now that it's ready to be approved?

Naybe `logo/tds_fdw.svg`?

I'd also would like to get @GeoffMontee review, as he is the lead developer and the original author of `tds_fdw` :-)

**Comment by juliogonzalez** [comment]
And if you want, feel free to add your name, or GitHub link, or both, to the README.md file to show that you are the original author of the logo :-)

Maybe you will also want to add that information to the SVG metadata (so far I see some metadata there such as the License, but not the author).

Anyway, not mandatory to get this PR accepted.

**Comment by RAFAELDEV2016** [comment]
@juliogonzalez Hi, I have renamed the SVG file according to your suggested filename.

File: logo/tds_fdw.svg [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   inkscape:version="1.0 (4035a4fb49, 2020-05-01)"
+   sodipodi:docname="FINAL.svg"
```
Comment (juliogonzalez) [inline logo/tds_fdw.svg:11]: Shouldn't this be changed to `tds_fdw.svg`?

Comment (RAFAELDEV2016) [reply logo/tds_fdw.svg:11]: It does not matter but it can be changed as well. I will update it.

Comment (RAFAELDEV2016) [reply logo/tds_fdw.svg]: Ok it is done ;-)

**Comment by juliogonzalez** [comment]
Since I could not get feedback from @GeoffMontee but he was OK with the general content of the previous logo, let's accept what we have now.

We can always adjust things later :-)

**Comment by GeoffMontee** [comment]
Hi @RAFAELDEV2016, 

Thanks for the contribution!

The logo has been added to the GitHub organization:

https://github.com/tds-fdw

**Pull request merged**